### ISSUE TITLE
Survey getting reset when getting message or call

### ISSUE THREAD

**Issue by naveenvignesh5** [surveyjs/survey-library#3460]
### Are you requesting a feature, reporting a bug or asking a question?
Reporting a bug

### What is the current behavior?
When surveyjs application is run on mobile browser and an incoming call comes the survey answers are reset and it restarts the survey.

### What is the expected behavior?
When surveyjs application is run on mobile browser and an incoming call comes the survey should be undisturbed.

### How would you reproduce the current behavior (if this is a bug)?
1. Run surveyjs application in mobile.
2. Call the mobile in the middle of taking a survey.

### Specify your

* browser: Chrome Mobile
* browser version: 94.0.4606.81
* surveyjs platform (angular or react or jquery or knockout or vue): React
* surveyjs version: ^1.8.22

**Comment by dmitrykurmanov** [comment]
I don't think that we could have access to phone calls via the js library. May be I am wrong. Could you please share with us some links to read about it ? Thanks.

**Comment by naveenvignesh5** [comment]
@dmitrykurmanov When running the surveyjs in mobile browser. In the middle of the survey if I receive a phone call, the surveyjs running in the mobile browser gets reset. Don't know why it happens as the browser application should be independent ?
